adafruit_wiznet5k/adafruit_wiznet5k.py

```diff
@@ -947,7 +1143,7 @@ def _write_socket(self, sock, address, data):
             )
         return None
 
-    def _read_socket(self, sock, address):
+    def _read_socket(self, sock: int, address: int) -> Optional[bytearray]:  # *1
```

Comment:
is the `# *1` comment something leftover or does it have some meaning for the types?